website/docs/r/compute_instance_volume.html.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: "scaleway"
+page_title: "Scaleway: scaleway_compute_instance_volume"
+sidebar_current: "docs-scaleway-resource-compute-instance-volume"
+description: |-
+  Manages Scaleway Compute Instance Volumes.
+---
+
+# scaleway_compute_instance_volume
+
+Creates and manages Scaleway Compute Instance Volumes. For more information, see [the documentation](https://developers.scaleway.com/en/products/instance/api/#volumes-7e8a39).
+
+## Example Usage
+
+```hcl
+resource "scaleway_compute_instance_volume" "server_volume" {
+    type       = "l_ssd"
+	name       = "some-volume-name"
+	size_in_gb = 20
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+- `type` - (Optional) Type type of the volume: `b_ssd` (Block SSD), `l_ssd` (Local SSD) or `l_hdd` (Local HDD). Defaults to `b_ssd`.
```

Comment:
Let's try to use the same format for showing defaut values / optional / required / oneof, this is different than [scaleway_compute_instance_server](https://github.com/terraform-providers/terraform-provider-scaleway/pull/193/files#diff-5c95b28312e3fd1bb991cb7cd9561937R98)

cc @QuentinBrosse

Reply:
Not sure if `l_hdd` should be documented

Reply:
> Let's try to use the same format

Yes, we should take [this syntax](https://github.com/terraform-providers/terraform-provider-scaleway/pull/193/files#diff-5c95b28312e3fd1bb991cb7cd9561937R143).

> Not sure if l_hdd should be documented

I'm agree

Reply:
> I'm agree

👋 Hi agree, nice to meet you.

Reply:
> 👋 Hi agree, nice to meet you.

Missed the opportunity to make a dad joke 😜